### ISSUE TITLE
warn: command 'extension.emmetMe' not found

### ISSUE THREAD

**Issue by kuncevic** [jakethashi/vscode-angular2-emmet#5]
Just getting the warn message `command 'extension.emmetMe' not found` when pressing the tab inside `ts` file, then the cursor just stuck without actually indenting the tab space. Had to disable the extension for now.
Seams to be same as this one https://github.com/Microsoft/vscode/issues/18642

**Comment by jakethashi** [comment]
Thanks for reporting this issue.

**Comment by honzajde** [comment]
I see this issue is still there vscode 1.13, angular-emmet 2.0.1

**Comment by jakethashi** [comment]
Can you write down step to reproduce the issue. Thanks.

**Comment by honzajde** [comment]
Strange, when I encountered this problem I had also Angular v4 Typescript Snippets extension enabled. Then I disabled them both and also issue has disappeared. Now I enabled them both, reloaded and I can not reproduce the issue. Now the question is: will it come back?

**Comment by jakethashi** [comment]
I am also unable to reproduce the issue. I guess it has been some odd state (which may occure in case the extension is not ready to use and vscode trying to execude it via tab key binding). In case You'll notice it again, please report the issue and try to describe steps to repreduce. Thanks/

**Comment by nimatullah** [comment]
I have the same issue.

**Comment by jakethashi** [comment]
Could you provide more details about vscode and emmet version?

**Comment by catchmareck** [comment]
I had the same issue after I updated to the latest version. Thanks to @honzajde I managed to fix it. I have these extensions installed: Angular 2 TypeScript Emmet, Angular v5 TypeScript Snippets, and TSLint. What I did to solve this issue was just disable-reload-enable-reload with each extension. Everything works fine now :)

**Comment by GoGoris** [comment]
Since I updated to the insiders build I am having the same issue when I enable both the 'Angular 2 typescript Emmet' plugin and the 'mikael.angular-beastcode' plugin. I have the following error inside the dev tools: "Activating extension 'jakethashi.vscode-angular2-emmet' failed: Cannot find module 'emmet/lib/utils/common'."

Vscode version:
Version: 1.26.0-insider
Commit: e8b1ee0b4d016bc38bbc1cf06d6d1868760d5fe8
Date: 2018-07-26T05:18:28.881Z
Electron: 2.0.5
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by mprinc** [comment]
Yes, restarting all angular enabled extensions and TSLint fixed the annoying problem. Exhausted of the problem to seek for real reason :)

**Comment by jckuhl** [comment]
I had this problem in a Vue project of all things.  I made a typescript file, not a .vue component a simple .ts file and it wouldn't let me tab until I disabled the extension.  Angular 2 Emmet 2.0.3 and VSCode 1.32.3

**Comment by engineerbaraazakariya** [comment]
same problem, fixed by: I just 'Disable all installed extensions' then a pop up showed to restarted Vscode with all extensions disabled I restarted it this popup, then a popup showed to restart with all extensions on again and I click on it and now the error vanished and things are working well again.
![Screen Shot 2022-04-04 at 5 48 24 PM](https://user-images.githubusercontent.com/5383782/161570444-79d96c63-b106-44bf-b238-83ea10babc01.png)
